app/src/main/kotlin/com/fernandocejas/sample/features/movies/MovieDetailsFragment.kt

```diff
@@ -100,10 +103,18 @@ class MovieDetailsFragment : BaseFragment() {
 
     private fun handleFailure(failure: Failure?) {
         when (failure) {
-            is NetworkConnection -> { notify(R.string.failure_network_connection); close() }
-            is ServerError -> { notify(R.string.failure_server_error); close() }
-            is NonExistentMovie -> { notify(R.string.failure_movie_non_existent); close() }
-            else -> { notify(R.string.failure_server_error); close() }
+            is NetworkConnection -> {
+                notify(R.string.failure_network_connection); close()
+            }
+            is ServerError -> {
+                notify(R.string.failure_server_error); close()
+            }
+            is NonExistentMovie -> {
+                notify(R.string.failure_movie_non_existent); close()
+            }
+            else -> {
+                notify(R.string.failure_server_error); close()
+            }
```

Comment:
Any reason why the style change? I guess it is a matter of taste. In my case is because I try to keep it concise but maybe I'm missing something :)